vault/identity_store_entities_test.go

```diff
@@ -1079,24 +1073,10 @@ func TestIdentityStore_MergeEntitiesByID(t *testing.T) {
 	}
 
 	entityID2 := resp.Data["id"].(string)
-	// Set entity ID in alias registration data and register alias
-	aliasRegisterData2["entity_id"] = entityID2
-
-	aliasReq = &logical.Request{
-		Operation: logical.UpdateOperation,
-		Path:      "alias",
-		Data:      aliasRegisterData2,
-	}
 
-	// Register the alias
-	resp, err = is.HandleRequest(ctx, aliasReq)
-	if err != nil || (resp != nil && resp.IsError()) {
-		t.Fatalf("err:%v resp:%#v", err, resp)
-	}
-
-	aliasRegisterData3["entity_id"] = entityID2
+	aliasRegisterData2["entity_id"] = entityID2
```

Comment:
Why did this test need to change?

Reply:
It was merging an entity that contained two aliases, one of which clashed. It made sense to remove the clash and keep it testing the happy path.

Reply:
Gotcha, ty.